pandas/tests/frame/test_combine_concat.py

```diff
@@ -930,6 +930,14 @@ def test_concat_datetime_datetime64_frame(self):
         # it works!
         pd.concat([df1, df2_obj])
 
+    def test_combine_first_empty_dataframe(self):
+        df1 = pd.DataFrame(columns=["a"])
```

Comment:
can you add the issue number here as a comment

Reply:
sure